src/components/Menu/index.tsx

```diff
@@ -11,48 +11,61 @@ import {
   MessageCircle,
   PieChart,
   Share2,
-  Triangle,
   UserPlus,
 } from 'react-feather'
-import { NavLink } from 'react-router-dom'
+import { NavLink, useLocation } from 'react-router-dom'
 import { useMedia } from 'react-use'
 import { Text } from 'rebass'
 import styled, { css } from 'styled-components'
 
 import { ReactComponent as BlogIcon } from 'assets/svg/blog.svg'
 import { ReactComponent as DiscoverIconSvg } from 'assets/svg/discover_icon.svg'
+import { ReactComponent as LightIcon } from 'assets/svg/light.svg'
 import { ReactComponent as RoadMapIcon } from 'assets/svg/roadmap.svg'
-import { ButtonPrimary } from 'components/Button'
+import { ButtonEmpty, ButtonPrimary } from 'components/Button'
+import { AutoColumn } from 'components/Column'
 import SlideToUnlock from 'components/Header/SlideToUnlock'
+import ArrowRight from 'components/Icons/ArrowRight'
 import Faucet from 'components/Icons/Faucet'
+import MailIcon from 'components/Icons/MailIcon'
+import LanguageSelector from 'components/LanguageSelector'
 import Loader from 'components/Loader'
 import MenuFlyout from 'components/MenuFlyout'
-import { AutoRow } from 'components/Row'
-import { ENV_LEVEL, ENV_TYPE, TAG } from 'constants/env'
-import { AGGREGATOR_ANALYTICS_URL, DMM_ANALYTICS_URL } from 'constants/index'
+import Row, { AutoRow } from 'components/Row'
+import NotificationModal from 'components/SubscribeButton/NotificationModal'
+import Toggle from 'components/Toggle'
+import ThemeToggle from 'components/Toggle/ThemeToggle'
+import { TutorialIds } from 'components/Tutorial/TutorialSwap/constant'
+import { TAG } from 'constants/env'
+import { AGGREGATOR_ANALYTICS_URL, APP_PATHS, DMM_ANALYTICS_URL } from 'constants/index'
+import { getLocaleLabel } from 'constants/locales'
 import { FAUCET_NETWORKS } from 'constants/networks'
 import { EVMNetworkInfo } from 'constants/networks/type'
 import { useActiveWeb3React } from 'hooks'
 import useClaimReward from 'hooks/useClaimReward'
 import useMixpanel, { MIXPANEL_TYPE } from 'hooks/useMixpanel'
 import useTheme from 'hooks/useTheme'
 import { ApplicationModal } from 'state/application/actions'
-import { useModalOpen, useToggleModal } from 'state/application/hooks'
+import { useModalOpen, useNotificationModalToggle, useToggleModal } from 'state/application/hooks'
+import { useTutorialSwapGuide } from 'state/tutorial/hooks'
+import { useDarkModeManager, useHolidayMode, useUserLocale } from 'state/user/hooks'
 import { ExternalLink } from 'theme'
+import { isChristmasTime } from 'utils'
 
 import ClaimRewardModal from './ClaimRewardModal'
 import FaucetModal from './FaucetModal'
 import NavDropDown from './NavDropDown'
 
 const MenuItem = styled.li`
   flex: 1;
-  padding: 0.75rem 0;
+  padding: 0.7rem 0;
```

Comment:
evil number

Reply:
tell me why

Reply:
0.7 * 16 = 11.2 px lol. Why would we need that little decimal px

Reply:
<img width="452" alt="image" src="https://user-images.githubusercontent.com/33005392/209562903-c090cb11-7fb7-433c-8f3f-1db1d0568bee.png">
lol, but I reverted.

Reply:
<img width="96" alt="image" src="https://user-images.githubusercontent.com/23650671/209563331-cb2f88ce-84de-454f-aa62-93ef0382fba6.png">

He missed `1`  maybe? :joy: